src/Features/CSharp/Portable/CodeRefactorings/PullMemberUp/CSharpPullMemberUpCodeRefactoringProvider.cs

```diff
@@ -31,8 +31,15 @@ public CSharpPullMemberUpCodeRefactoringProvider() : base(null)
         protected override async Task<SyntaxNode> GetSelectedNodeAsync(Document document, TextSpan span, CancellationToken cancellationToken)
         {
             var refactoringHelperService = document.GetLanguageService<IRefactoringHelpersService>();
-            return await refactoringHelperService.TryGetSelectedNodeAsync(
-                document, span, n => n is MemberDeclarationSyntax || n is VariableDeclaratorSyntax, cancellationToken).ConfigureAwait(false);
+
+            var memberDecl = await refactoringHelperService.TryGetSelectedNodeAsync<MemberDeclarationSyntax>(document, span, cancellationToken).ConfigureAwait(false);
+            if (memberDecl != default)
```

Comment:
nit: in general, we use 'null' to compare reference types.  But we're not hugely pedantic here.  If you prefer this, it's totally reasonable to ask for it.

Reply:
I'm pretty sure I saw that somewhere else in the codebase so I just assumed it's the preferred style. Will switch from now on.

Reply:
it is for certain syntax constructs like tokens (since they are structs).  Note: i dont' have a strong opinion here.  But `is object` or `is null` or `== null` or `!= null` are *deeply* idiomatic for C#.  So they tend to eb preferred.